### PR TITLE
missing romeo.urdf

### PR DESCRIPTION
**Pull request by nlyubova**
Could we add the romeo.urdf ?

### CODE REVIEW

**Comment by vrabaud** [comment]
@Karsten1987 , what were the problems in loading the urdf from the proper *_description packages ?

**Comment by Karsten1987** [comment]
It's only for qibuild purposes. 
Have a look in here:
https://github.com/ros-naoqi/naoqi_driver/blob/master/src/helpers/filesystem_helpers.hpp#L115

We could replace the rospack call from naoqi_driver to nao_description, pepper_description etc, but we'd still have to keep a local copy of each URDF when building with qibuild.

**Comment by Karsten1987** [comment]
@nlyubova could you please verify that the joint_state publisher works fine? Just launch it with a ROMEO and check if you see it in RViz. 
Have a look at https://github.com/ros-naoqi/naoqi_driver/blob/master/src/converters/joint_state.cpp and also at the according files for sonar.

**Comment by nlyubova** [comment]
@Karsten1987 , that would be great to replace the rospack call from naoqi_driver to robot_description but we should also pass the BodyVersion for the path, like "pepper1.0_generated_urdf" or "naoV40_generated_urdf"

For testing, I've made a romeo_bringup and it works well with the naoqi_driver except of calls to ALBodyTemperature and ALAutonomousLife.

**Comment by Karsten1987** [comment]
@nlyubova  The problems with the fingers got resolved in here https://github.com/ros-naoqi/naoqi_driver/pull/42

For the rest can this be merged? I cannot verify this PR remotely.

**Comment by vrabaud** [comment]
@nlyubova , please rebase and push to your repo, you have way too many changes

File: src/converters/diagnostics.cpp [hunk 1]
```diff
@@ -54,12 +54,12 @@ namespace converter
...
     BaseConverter( name, frequency, session ),
     p_memory_(session->service("ALMemory")),
-    p_body_temperature_(session->service("ALBodyTemperature")),
+    //p_body_temperature_(session->service("ALBodyTemperature")),
     temperature_warn_level_(68),
     temperature_error_level_(74)
 {
   // Allow for temperature reporting (for CPU)
-  p_body_temperature_.call<void>("setEnableNotifications", true);
+  //p_body_temperature_.call<void>("setEnableNotifications", true);
```
Comment (vrabaud) [inline src/converters/diagnostics.cpp]: disable only if it is romeo so replace your comment by something like:
```
if ((robot_ == robot::PEPPER) || (robot_ == robot::NAO)) {
  p_body_temperature_ = session->service("ALBodyTemperature");
  p_body_temperature_.call<void>("setEnableNotifications", true);
}
```

File: src/helpers/filesystem_helpers.hpp [hunk 2]
```diff
@@ -113,6 +113,8 @@ inline std::string& getURDF( std::string filename )
 {
 #ifdef CATKIN_BUILD
   static std::string path = ros::package::getPath("naoqi_driver")+"/share/urdf/"+filename;
+  //static std::string path = ros::package::getPath(filename+"_description")+"/urdf/"+filename+".urdf";
```
Comment (vrabaud) [inline src/helpers/filesystem_helpers.hpp]: this comment is useless

File: src/converters/joint_state.cpp [hunk 3]
```diff
@@ -88,7 +88,6 @@ void JointStateConverter::callAll( const std::vector<message_actions::MessageAct
   // get joint state values
   std::vector<double> al_joint_angles = p_motion_.call<std::vector<double> >("getAngles", "Body", true );
   const ros::Time& stamp = ros::Time::now();
-
```
Comment (vrabaud) [inline src/converters/joint_state.cpp]: please do not change the file

File: src/converters/diagnostics.hpp [hunk 4]
```diff
@@ -64,7 +64,7 @@ class DiagnosticsConverter : public BaseConverter<DiagnosticsConverter>
   /** Proxy to ALMemory */
   qi::AnyObject p_memory_;
   /** Proxy to ALBodyTemperature */
-  qi::AnyObject p_body_temperature_;
+  //qi::AnyObject p_body_temperature_;
```
Comment (vrabaud) [inline src/converters/diagnostics.hpp]: leave this one as is as you will handle Romeo differently

File: src/converters/diagnostics.cpp [hunk 5]
```diff
@@ -54,12 +54,15 @@ namespace converter
 DiagnosticsConverter::DiagnosticsConverter( const std::string& name, float frequency, const qi::SessionPtr& session ):
     BaseConverter( name, frequency, session ),
     p_memory_(session->service("ALMemory")),
-    p_body_temperature_(session->service("ALBodyTemperature")),
+    //p_body_temperature_(session->service("ALBodyTemperature")),
```
Comment (vrabaud) [inline src/converters/diagnostics.cpp]: please, remove this line as you initialize below

**Comment by vrabaud** [comment]
The romeo urdf has a lot of personnalization so I am against it.

**Comment by vrabaud** [comment]
can you please not commit the URDF of  your Romeo so that we can merge that commit ? Thx.

**Comment by nlyubova** [comment]
Ok, that was for the demo, sorry for that

**Pull request merged**